bin/kafka-server-stop.sh

```diff
@@ -14,7 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 SIGNAL=${SIGNAL:-TERM}
-PIDS=$(ps ax | grep -i 'kafka\.Kafka' | grep java | grep -v grep | awk '{print $1}')
+PIDS=$(ps ax | grep -i 'kafkaServer' | grep java | grep -v grep | awk '{print $1}')
```

Comment:
That would catch the Confluent distro of Kafka as well as the CDH one, which may or may not be desired.  It's also only matching on a GC log file name as far as I can tell, which doesn't seem ideal.